features/support/env.rb

```diff
@@ -4,6 +4,12 @@
 # instead of editing this one. Cucumber will automatically load all features/**/*.rb
 # files.
 
+require 'simplecov'
+SimpleCov.start
```

Comment:
According to https://github.com/colszowka/simplecov I believe this should be `SimpleCov.start 'rails'`

Reply:
That works too. I just pulled this snippet from https://github.com/codecov/codecov-ruby#usage